indexer/cli/cli.go

```diff
@@ -32,6 +35,11 @@ func runIndexer(ctx *cli.Context) error {
 	return nil
 }
 
+// Maybe make NewDB take a config.DBConfig instead of a string in future cleanup
+func getDsn(dbConf config.DBConfig) string {
+	return fmt.Sprintf("postgresql://%s:%s@%s:%s/%s", dbConf.User, dbConf.Password, dbConf.Host, dbConf.Port, dbConf.Name)
```

Comment:
```suggestion
	return fmt.Sprintf("postgresql://%s:%s@%s:%s/%s", dbConf.User, dbConf.Password, dbConf.Host, strconv.Itoa(dbConf.Port), dbConf.Name)
```
Noticed the linter thorwing a warning for this